deployments/salt/Makefile

```diff
@@ -10,4 +10,13 @@ run-dev-image:
 		--name signalfx-agent-salt-dev \
 		-it \
 		signalfx-agent-salt-dev \
-		bash
\ No newline at end of file
+		bash
+
+.PHONY: test
+test:
+	docker run --rm signalfx-agent-salt-dev \
```

Comment:
Make this run the commands directly, not through docker.  You can add instructions on running a docker dev image to the README for salt if it isn't already there.